server/src-lib/Hasura/Backends/Postgres/Connection.hs

```diff
@@ -168,12 +185,12 @@ mkTxErrorHandler isExpectedError txe = fromMaybe unexpectedError expectedError
         PGIntegrityConstraintViolation code ->
           let cv = (ConstraintViolation,)
               customMessage = (code ^? _Just._PGErrorSpecific) <&> \case
-                PGRestrictViolation   -> cv "Can not delete or update due to data being referred. "
-                PGNotNullViolation    -> cv "Not-NULL violation. "
+                PGRestrictViolation -> cv "Can not delete or update due to data being referred. "
+                PGNotNullViolation -> cv "Not-NULL violation. "
                 PGForeignKeyViolation -> cv "Foreign key violation. "
-                PGUniqueViolation     -> cv "Uniqueness violation. "
-                PGCheckViolation      -> (PermissionError, "Check constraint violation. ")
-                PGExclusionViolation  -> cv "Exclusion violation. "
+                PGUniqueViolation -> cv "Uniqueness violation. "
+                PGCheckViolation -> (PermissionError, "Check constraint violation. ")
+                PGExclusionViolation -> cv "Exclusion violation. "
```

Comment:
nit: could you run stylish-haskell on this PR? It would reduce the amount of changes in this PR and be nicer overall. ^^

Reply:
Done.